cmd/cluster/pitr-config/pitr_config_ops.go

```diff
@@ -97,15 +149,35 @@ var createPitrConfigCmd = &cobra.Command{
 			logrus.Debugf("Full HTTP response: %v", r)
 			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
 		}
+		pitrConfigId := resp.Data.Info.Id
 
-		fmt.Printf("\nSuccessfully created PITR configuration.\n\n")
+		msg := fmt.Sprintf("The PITR Configuration for namespace %s in cluster %s is being created\n\n", formatter.Colorize(namespaceName, formatter.GREEN_COLOR), formatter.Colorize(ClusterName, formatter.GREEN_COLOR))
 
-		pitrConfigCtx := formatter.Context{
-			Output: os.Stdout,
-			Format: formatter.NewPitrConfigFormat(viper.GetString("output")),
-		}
+		if viper.GetBool("wait") {
```

Comment:
This code block is being repeated at multiple places, can we extract it into a helper function?

Reply:
Something like this perhaps?
`func handleTaskCompletion(authApi *ybmAuthClient.AuthApi, clusterID, msg string, taskType ybmclient.TaskTypeEnum)`